editor/src/components/canvas/canvas-strategies/interaction-state.ts

```diff
@@ -340,6 +338,32 @@ export function hasDragModifiersChanged(
   )
 }
 
+export function hasKeyboardModifierChanged(
```

Comment:
it looks like this function is not used anywhere, only imported in dispatch-strategies.tsx